frontend/pages/dataset/_id/annotation-mode/index.vue

```diff
@@ -7,8 +7,9 @@
           :key="datasetId"
           :datasetId="datasetId"
           :breadcrumbs="breadcrumbs"
-          :showTrainButton="isAdminOrOwnerRole"
```

Comment:
Be careful Leire, this commit is changing the behaviour of the component and some tests are failing.
Why is this component receiving props that it didn't need before?